File: coderpad/start.exs
ExUnit.start

defmodule Example do
  def calculate(num) do
    div(num, 3) - 2
  end
end

defmodule ExampleTest do
  use ExUnit.Case

  test "calculates expression" do
    assert 2 = Example.calculate(12)
    assert 2 = Example.calculate(14)
    assert 654 = Example.calculate(1969)
    assert 33583 = Example.calculate(100756)
  end
end

File: coderpad/start.rb
require 'rspec/autorun'

class Example
  def calculate(num)
    num / 3 - 2
  end
end

RSpec.describe Example do
  it 'calculates expression' do
    example = Example.new

    expect(example.calculate(12)).to eq(2)
    expect(example.calculate(14)).to eq(2)
    expect(example.calculate(1969)).to eq(654)
    expect(example.calculate(100756)).to eq(33583)
  end
end

